scripts/ci/run-ci-tests.sh

```diff
@@ -63,8 +63,9 @@ ci_prep () {
 
 test_stream() {
 	# Testing CRIU streaming to criu-image-streamer
+	STREAM_TEST_EXCLUDE="-x maps09 -x maps10"
```

Comment:
@minhbq-99 Would it be possible to add a comment (e.g., FIXME or TODO) that explains why we skip these tests?

https://github.com/checkpoint-restore/criu/pull/1622#issuecomment-1002355360